services/API-service/src/api/data/sql/get-country-event.sql

```diff
@@ -0,0 +1,37 @@
+select
+  to_char(max(end_date) , 'yyyy-mm-dd') as end_date,
+  to_char(min(start_date), 'yyyy-mm-dd') as start_date,
+  max(country_code) as country_code,
+  max(active_trigger::int)::boolean as active_trigger
+from
+  (
+  select
+    e.pcode,
```

Comment:
`e.pcode as placeCode` instead of `e.pcode`

and other appropriate aliases for readability and consistency

aliases preferably in [camelCase](https://news.ycombinator.com/item?id=6777600) for use in `TS`